install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1602,9 +1602,21 @@ localRedirectPolicy: false
 # To include or exclude matched resources from cilium identity evaluation
 # labels: ""
 
-# logOptions allows you to define logging options. eg:
-# logOptions:
-#   format: json
+# -- (string) Allows you to define logging options.
+# LogOpt sets log driver options for cilium
```

Comment:
What is `LogOpt`? Don't you mean `logOptions`?

Reply:
Yes, logOptions. It's named/called differently in other parts of the code, another copy/pasta issue. I'll correct this.